<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLotDialog</class>
 <widget class="QDialog" name="AddLotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_code">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_code"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_description"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_production">
     <property name="text">
      <string>Production Date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_production"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_expiration">
     <property name="text">
      <string>Expiration Date</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_expiration"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
